src/Forms/FormField.php

```diff
@@ -769,17 +770,34 @@ public function attrValue()
     /**
      * Set the field value.
      *
-     * @param mixed $value
-     * @param null|array|DataObject $data {@see Form::loadDataFrom}
+     * If a FormField requires specific behaviour for loading content from either the database
+     * or a submitted form value they should override setSubmittedValue() instead.
+     *
+     * @param mixed $value Either the parent object, or array of source data being loaded
+     * @param array|DataObject $data {@see Form::loadDataFrom}
      * @return $this
      */
-    public function setValue($value)
+    public function setValue($value, $data = null)
```

Comment:
Not sure where forms.md is; I'll have a look at refreshing the forms docs dir though.